src/Layouts/stars/feed/note.tsx

```diff
@@ -0,0 +1,108 @@
+import { ReplyOutlined, DeleteOutline } from "@material-ui/icons";
+import { Grid, Avatar, Typography, Divider } from "@mui/material";
+import { useState } from "react";
+import { noteType } from "../../../assets/star";
+
+interface noteProps {
```

Comment:
instead of  `| undefined`  you and write `key?: type` and ts will do the same (more readable . bit dickish move to comment)

Reply:
the problem is later in this file I have to send undefined as a parameter to setReplyTo in order for my logic to work